test/romloader.t.cpp

```diff
@@ -47,11 +47,14 @@ TEST_F (RomLoaderTest, loadValidFile)
     bool isLoaded = romLoader.load(fileName);
 
     IMemory::CartridgeData data = romLoader.getData();
-    for (int i = 0; i < 512; i++) {
+    for (uint16_t i = 0x0000; i < 0x0100; i++) {
```

Comment:
sa,e here no magic numbers

Reply:
can't find why i didi this putting it in TODO